inference/gradio_openchatkit.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# @Desc   : openchat kit gradio application
+"""
+            Run:
+                # under OpenChatKit/inference from https://github.com/togethercomputer/OpenChatKit
+                CUDA_VISIBLE_DEVICES=2,3 python3 gradio_openchatkit.py
+            Warn:
+                the bigger max_new_tokens the more cuda mem, so be careful
+"""
+
+import os
+import sys
+
+CUR_DIR = os.path.abspath(os.path.dirname(__file__))
+MODEL_PATH = os.path.join(CUR_DIR, "../../GPT-NeoXT-Chat-Base-20B/")
```

Comment:
I believe this should be 

```suggestion
MODEL_PATH = os.path.join(CUR_DIR, "../huggingface_models/GPT-NeoXT-Chat-Base-20B/")
```